kodu main sınıfında çalıstırıyoruz ama demo error_handling de 
o yüzden hata alıyoruz 
şimdi main sınıfına atıp çalıştıgını görecez